modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectViewLocalServiceTest.java

```diff
@@ -99,7 +99,7 @@ public void setUp() throws Exception {
 	@Test
 	public void testAddObjectView() throws Exception {
 		_objectViewLocalService.addObjectView(
-			TestPropsValues.getUserId(),
+			RandomTestUtil.randomString(), TestPropsValues.getUserId(),
```

Comment:
It would be nice to assert the externalReferenceCode used to add the ObjectView. Please see this [example](https://github.com/liferay/liferay-portal/blob/5363ba79d6f5607b99bc803ceb6c7b50ff779058/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java#L1875-L1876)